frontendIcons/src/main/java/icons/MacheteIcons.java

```diff
@@ -17,6 +17,8 @@ private MacheteIcons() {}
 
   public static final Icon MACHETE_FILE = IconLoader.getIcon("/macheteFileIcon.svg");
```

Comment:
Btw while you're here: I'd maybe use some icon with a paper & pencil (and not just paper) to indicate it's about editing, not e.g. just viewing

Reply:
IMHO this action itself is just about opening the file. 
I could not find such an icon within IntelliJ icons.
Forwarded to #326.